unittests/InterOp/ScopeReflectionTest.cpp

```diff
@@ -554,6 +555,7 @@ TEST(ScopeReflectionTest, GetBaseClassOffset) {
   EXPECT_EQ(InterOp::GetBaseClassOffset(S, Decls[3], Decls[0]), 0);
   EXPECT_EQ(InterOp::GetBaseClassOffset(S, Decls[3], Decls[1]), 4);
   EXPECT_EQ(InterOp::GetBaseClassOffset(S, Decls[3], Decls[2]), 8);
+  EXPECT_EQ(InterOp::GetBaseClassOffset(S, Decls[4], Decls[0]), 12);
```

Comment:
Why here it is 12?

Reply:
https://godbolt.org/z/3db7M9sM6

I don't know why but it checks out

Reply:
Ok, fair enough...